### DISCUSSION TITLE
Live/Real time transcription from laptop mic

### DISCUSSION THREAD

**Discussion by sb1992** [deepgram/community#432] [General help] [answered]
Hi Folks

I was wondering how do i connect my laptop's mic and do a real time transcription a bit similar to what is also shown in the demo?

**Comment by lukeocodes** [comment, marked as answer]
Hi @sb1992 

We have this example: https://github.com/deepgram-devs/js-live-example

It does have a small node script, which is responsible for generating a limited scope temporary API key for the frontend client to use. If this is for localhost use only, you COULD bypass that. But, we do not recommend putting your API key in the client.
